src/DynamoManipulation/NodeManipulator.cs

```diff
@@ -65,9 +66,9 @@ public abstract class NodeManipulator : INodeManipulator
         /// <summary>
         /// Returns list of Gizmos used by this manipulator.
         /// </summary>
-        /// <param name="createIfNone">Whether to create new gizmo if not already present.</param>
+        /// <param name="createOrUpdate">Whether to create new gizmo or to update an existing one.</param>
         /// <returns>List of Gizmos.</returns>
-        protected abstract IEnumerable<IGizmo> GetGizmos(bool createIfNone);
+        protected abstract IEnumerable<IGizmo> GetGizmos(bool createOrUpdate);
```

Comment:
I changed it to `createOrUpdate` is because I want to avoid creating or updating the gizmos when all I need to do is query for them. Will update the documentation accordingly.